karma.shared.js

```diff
@@ -2,16 +2,19 @@
 
 /* This is the shared karma config */
 
+var merge = require('lodash.merge');
 var RewirePlugin = require('rewire-webpack');
-var webpackConfig = require('./webpack.config.js');
+var webpackConfig = require('./webpack.config');
 
+
+var newWebpackConfig = merge({}, webpackConfig);
```

Comment:
@kumar303 actually assign isn't going to work for this. See http://jsbin.com/nuhawi/edit?js,console for why.